lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -328,3 +351,30 @@ def test_query_counts(self):
             )
             self.assertEqual(resp.status_code, status.HTTP_200_OK)
             self.assertEqual(len(resp.data), 2)
+
+    @patch.dict(settings.FEATURES, {'CERTIFICATES_HTML_VIEW': True})
+    def test_with_no_certificate_configuration(self):
+        """
+        Verify that certificates are not returned until there is no active
+        certificate configuration.
+        """
+        self.cert.download_url = ''
+        self.cert.save()
+
+        response = self.get_response(
+            AuthType.jwt,
+            requesting_user=self.student,
+            requested_user=self.student,
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        self.assertEqual(resp.data, [])
```

Comment:
Here, I think your intent is 'response' rather than 'resp'.

Reply:
Resolved.